stable-profiles/0.9.52/firejailed-tor-browser.profile

```diff
@@ -0,0 +1,117 @@
+# Firejail profile for firejailed-tor-browser
+# This file is overwritten after every install/update
+# Persistent local customizations
+include ${HOME}/.config/firejail/firejailed-tor-browser.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+# Note: PluggableTransports didn't work with this profile
+
+noblacklist ${HOME}/.firejailed-tor-browser
+noblacklist ${HOME}/.config/firejail
+
+blacklist /opt
+blacklist /run/dbus/system_bus_socket
+blacklist /srv
+blacklist /usr/games
+blacklist /usr/local
+blacklist /usr/src
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
+
+whitelist ${HOME}/.firejailed-tor-browser
+whitelist ${HOME}/.config/firejail
```

Comment:
Why?

Reply:
Because I noticed that these statements where included in the default `/etc/firejail/torbrowser-launcher.profile` which I used as a template for creating my own profile for my specific environment.

Reply:
To clarify further, the following statements where not included in the default `/etc/firejail/tor-browser-en.profile` or in `/etc/firejail/torbrowser-launcher.profile` but I did notice them in other program profiles which I thought (at the time) might be a good idea to include/blacklist.
```
blacklist /opt
blacklist /run/dbus/system_bus_socket
blacklist /srv
blacklist /usr/games
blacklist /usr/local
blacklist /usr/src
```
Perhaps these statements are not necessary. However, it currently works for my setup. I have no argument against removing those statements. I shall I remove them?

Reply:
I just remembered why I included this block of blacklisted statements. It was because I noticed that they where included in the firejailed-tor-browser.profile for version 0.9.58. So I thought it might be a good idea to try it for creating a profile for version 0.9.52. Either way, the profile works with or without this block of blacklisted statements for version 0.9.52. Again, I will defer back to you. What do you recommend?

Reply:
I only talk about `noblacklist ${HOME}/.config/firejail`/`whitelist ${HOME}/.config/firejail`, but GH always shows the two lines above.

Reply:
Ok I comprehend. Thanks for clarifying.